src/styles/colors.tsx

```diff
@@ -23,8 +24,6 @@ export enum Colors {
   goldBrand = '#FBCC5C',
   /** @deprecated */
   onboardingBrownLight = '#A49B80',
```

Comment:
Is `onboardingBrownLight` still used or could we remove it? It looks like it was removed in a lot of the files above

Reply:
I swapped it out in some of the onboarding screens; however, it's still used in several places: [Slack Thread](https://valora-app.slack.com/archives/C03D3K379FE/p1719612125810649).